.github/workflows/manual-nixpkgs-v2.yml

```diff
@@ -38,4 +36,4 @@ jobs:
           authToken: '${{ secrets.CACHIX_AUTH_TOKEN }}'
 
       - name: Building Nixpkgs manual
-        run: NIX_PATH=nixpkgs=$(pwd)/untrusted nix-build --option restrict-eval true untrusted/ci -A manual-nixpkgs -A manual-nixpkgs-tests
```

Comment:
Seems this isn't having an effect on _this_ PR's build.

https://github.com/NixOS/nixpkgs/actions/runs/15326984888/job/43123920179?pr=412102#step:6:46

I assume because this is a `pull_request_target` workflow, so the workflow file used to run CI on _this_ PR is the one without this change.

I guess if we want to test this PR effectively we need to land this change separately or test exclusively in forks?